website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,268 @@
+})
+...
+```
+
+In the above example, if a CR(CustomResource), which is managed by its controllers, is applied to a Namespace that is not in this List of Namespaces then, its controller will not re-trigger its reconcile since the Namespaced is NOT part of the [Manager][ctrl-manager] scope.
+
+**IMPORTANT:** Note that this is not intended to be used for excluding Namespaces, this is better done via a Predicate. Also note that you may face performance issues when using this with a high number of Namespaces.
+
+## Restricting Roles and permissions
+
+The operator-scope defines the [Manager][ctrl-manager] cache scope but not the permissions to access the resources. After updating the Manager's scope to be Namespaced, the [Kubernetes Role-Based Access Control or the (RBAC)][k8s-rbac] permissions should be restricted accordingly.
+
+These permissions are found in the directory `config/rbac/`. The `ClusterRole` and `ClusterRoleBinding` manifests are responsible for setup the permissions which allow the operator to have access to the resources.
+
+**NOTE** The other files that can be found in this directory not requires changes to achieve this goal. The RBAC manifests with the names `<kind>_editor_role.yaml` and `<kind>_viewer_role.yaml` are not used by the operator and are meant to be helpers for specific config to allow access to users and clients per Kind. Also, the files with the names `auth_proxy_*.yaml` are specific to the [kube-auth-proxy][kube-rbac-proxy]. 
+
+### Changing the permissions 
+
+To change the scope of the RBAC permissions from cluster-wide to a specific namespace, you will need to use a `Role` instead of `ClusterRole` and `RoleBinding` instead of c
+ 
+[`RBAC markers`][rbac-markers] defined in the controller (e.g `controllers/cache/memcached_controller.go`)
+are used to generate the operator's [RBAC ClusterRole][rbac-clusterrole] (e.g `config/rbac/role.yaml`). The default markers don't specify a `namespace` property and will result in a ClusterRole.
+
+Update the RBAC markers to specify a `namespace` property so that `config/rbac/role.yaml` is generated as a `Role` instead of a `ClusterRole`.
+
+Replace:
+
+```go
+// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds/status,verbs=get;update;patch
+```
+
+With the Namespace `marker`:
+
+```go
+// +kubebuilder:rbac:groups=cache.example.com,namespace="my-namespace",resources=memcacheds,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=cache.example.com,namespace="my-namespace",resources=memcacheds/status,verbs=get;update;patch
+```
+
+And then, run `make manifests` to update `config/rbac/role.yaml`:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+...
+```
+
+<!-- todo(camilamacedo86): remove the note or improve the above info when we have further info in the issue tracked  -->     
```

Comment:
This may have been due to a typo: https://github.com/kubernetes-sigs/kubebuilder/issues/1496#issuecomment-631160626.

Reply:
unfortunately, it is not. Info added in the issue.

Reply:
Since ^ is not a bug I recommend removing this section and instead say
```
We also need to update our ClusterRoleBindings to RoleBindings since they are not regenerated
```

Reply:
I still believe that it is a BUG but I will leave the comment and the step as suggested.